unit-tests/live/metadata/test-metadata-alive.py

```diff
@@ -0,0 +1,112 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2023 Intel Corporation. All Rights Reserved.
+
+# test:device D400*
+# test:device each(D500*)
```

Comment:
why the difference between d400and d500?

Reply:
use each for both please
We will also need to disable the run on d500 corrently until it will be available on LibCI.
Maybe this will work
after the each (D500*)
#donotrun D500*